code/modules/mining/machine_stacking.dm

```diff
@@ -7,6 +7,7 @@
 	desc = "Controls a stacking machine... in theory."
 	density = FALSE
 	anchored = TRUE
+	circuit = /obj/item/circuitboard/machine/stacking_unit_console

```

Comment:
You mind fixing the problem with this? below `speed_process` is set to `TRUE`. This machine doesn't need it; it's just a console.

That said, `/obj/machinery/mineral/stacking_machine` hypothetically does (though I'm not sure, anymore, what with `HasProximity`